applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -637,6 +655,15 @@ pub async fn command_runner(
                     Err(e) => eprintln!("BurnTari error! {}", e),
                 }
             },
+            CreateKeyCombo(args) => match create_key_combo(key_manager_service.clone(), args.key_seed).await {
+                Ok(KeyComboPair { sk, pk }) => {
+                    debug!(
```

Comment:
```suggestion
                    println!(
```

can change more than a single line with the suggestion.

Reply:
Great, slipped my fingers !